src/main/java/org/minions/devfund/chelo/Squarelotron.java

```diff
@@ -0,0 +1,183 @@
+/**
+ * Class Squarelotron.
+ */
+public class Squarelotron {
+
+    protected int[][] squarelotron;
+    protected int size;
+
+    /**
+     * Contructor for Squarelotron.
+     * @param n size of the matrix
+     */
+    public Squarelotron(int n) {
+        this.size = n;
+        this.squarelotron = new int[n][n];
+        int counter = 1;
+        for (int i = 0; i < n; i++) {
+            for (int j = 0; j < n; j++) {
+                this.squarelotron[i][j] = counter;
+                counter++;
+            }
+        }
+    }
+
+    /**
+     * A method to fill the 2-dimensional array with the numbers 1 to n squared.
+     * @param n size of the array
+     * @return array squarelotron filled
+     */
+    Squarelotron initializeArray(int n) {
+        Squarelotron array = new Squarelotron(n);
```

Comment:
Here you are creating another instance.
Please remove that!

Reply:
Some scripts fails if I remove that